lib/opencensus/trace/span_context.rb

```diff
@@ -287,6 +256,35 @@ def contained_span_builders
           builders.find_all { |sb| contains? sb.context.parent }
         end
       end
+
+      private
+
+      ##
+      # Create a child of this SpanContext, with a random unique span ID.
+      #
+      # @return [SpanContext] The created child context.
+      # @private
```

Comment:
nit: these don't need to be annotated private anymore if they are truly private.

Same for `get_span`